### DISCUSSION TITLE
Can't add mods from curseforge

### DISCUSSION THREAD

**Discussion by Oddfrogg213** [gorilla-devs/ferium#335] [Q&A] [answered]
Whenever I try to add a new mod form curseforge I get the following error code:

Adding mod... error decoding response body: unknown field `latestEarlyAccessFilesIndexes`, expected one of `id`, `gameId`, `name`, `slug`, `links`, `summary`, `status`, `downloadCount`, `isFeatured`, `primaryCategoryId`, `categories`, `classId`, `authors`, `logo`, `screenshots`, `mainFileId`, `latestFiles`, `latestFilesIndexes`, `dateCreated`, `dateModified`, `dateReleased`, `allowModDistribution`, `gamePopularityRank`, `isAvailable`, `thumbsUpCount` at line 1 column 13372

Any help would be much appreciated

**Comment by theRookieCoder** [comment, marked as answer]
This would be a bug in furse that I will get to soon.
